src/services/FileSystemService.real.spec.ts

```diff
@@ -228,7 +228,7 @@ describe('FileSystemService - REAL', () => {
       await expect(fileSystemService.createDirectory(mockFilePath)).rejects.toThrow('EEXIST');
     });
 
-    it('throws an error if the target is a broken symbolc link', async () => {
+    it.skip('throws an error if the target is a broken symbolc link', async () => {
```

Comment:
Add comment whit it is skipped.

Reply:
Fixed in b4c96e4c7244afcd90bac9258a1241bb26cc4304